kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java

```diff
@@ -79,16 +82,16 @@ public void updateTopic() {
                 .setMaxMessageBytes(UPDATED_MAX_MESSAGE_BYTES)
                 .sendData()
                 .waitUntilScreenReady();
-
         pages.openTopicsList(CLUSTER_NAME)
                 .waitUntilScreenReady();
         pages.openTopicView(CLUSTER_NAME, TOPIC_TO_UPDATE)
-                .openEditSettings()
-                // Assertions
-                .cleanupPolicyIs(COMPACT_POLICY_VALUE)
-                .timeToRetainIs(UPDATED_TIME_TO_RETAIN_VALUE)
-                .maxSizeOnDiskIs(UPDATED_MAX_SIZE_ON_DISK)
-                .maxMessageBytesIs(UPDATED_MAX_MESSAGE_BYTES);
+                .openEditSettings();
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(COMPACT_POLICY_VALUE).isEqualTo(pages.topicsList.openTopic(TOPIC_TO_UPDATE).openEditSettings().cleanupPolicyIs(COMPACT_POLICY_VALUE));
```

Comment:
remove please 'pages.topicsList.openTopic(TOPIC_TO_UPDATE).openEditSettings()' from assertions

Reply:
@ArthurNiedial why do all soft assertions from this method use regular assertions from TopicCreateEditSettingsView.class. Does SoftAssertions library really works this way?

Reply:
As we agree i plased "TODO" to refactor assertions to settings tab using softassertions .

Reply:
UPD: issue causing test were disabled resolved. need to update assertions